src/android/com/adobe/phonegap/push/GCMIntentService.java

```diff
@@ -236,7 +239,7 @@ private Bundle normalizeExtras(Context context, Bundle extras) {
                     String notifkey = iterator.next();
 
                     Log.d(LOG_TAG, "notifkey = " + notifkey);
-                    String newKey = normalizeKey(notifkey);
+                    String newKey = normalizeKey(notifkey, null, null);
```

Comment:
Because in this case occures normalization of the another type of notifications with another structure. But not sure if it's right.
Just in case i added parameters for this case too